### PR TITLE
Openstack glance

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by toabctl** [comment]
The corresponding OBS project which uses these files already is https://build.opensuse.org/project/show/home:tbechtold:openstack-containers:j2gen:SLE_15_SP1

There are still open questions:
- what should be the version of the openstack-services-base-image ? Currently I use the version from the `python3-oslo.utils` rpm package (which is obviously not good)
- naming of the containers. Do we need a product prefix?
- labels? Are theses correct? I guess we miss some...

File: openstack-glance-image/openstack-glance-image.kiwi.j2 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      </containerconfig>
+    </type>
+    {{ image.preferences.basics }}
+  </preferences>
+  {{ image.repository }}
+  <packages type="image">
+    <package name="openstack-glance"/>
+    <package name="openstack-glance-api"/>
+    <package name="openstack-glance-registry"/>
+    <package name="python3-glance"/>
```
Comment (dirkmueller) [inline openstack-glance-image/openstack-glance-image.kiwi.j2]: should be a dependency? don't list it?

Comment (toabctl) [reply openstack-glance-image/openstack-glance-image.kiwi.j2]: Done

**Comment by JanZerebecki** [comment]
Will not be used any more.

**Pull request closed without merging**